lesson_27/tests/parcel_tracking.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from webdriver_manager.chrome import ChromeDriverManager
+from lesson_27.pages.tracking_page import TrackingPage
+
+
+@pytest.fixture
+def setup_browser():
+    driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()))
+    driver.maximize_window()
+    yield driver
+    driver.quit()
+
+
+class TestTrackingStatus:
+    def test_tracking_status_found(self, setup_browser):
+        driver = setup_browser
+        driver.get("https://tracking.novaposhta.ua/#/uk")
```

Comment:
this should be in open() method
there should be no driver.get, driver.find_element etc.. in tests, only user actions